### DISCUSSION TITLE
Tree without displaying the root?

### DISCUSSION THREAD

**Discussion by bgyarfas** [Textualize/rich#1463] [Q&A] [answered]
I'm using textual to create an interactive interface for pyinvoke. I'm using a Tree to display the various tasks/collections but I would like to not display the root to save on screen space and since it's not adding any information (for my usage). A faked example of what I'm looking for... instead of this:
```
>>> tree = Tree("Rich Tree")
>>> tree.add("foo")
>>> tree.add("bar")
>>> print(tree)

Rich Tree
├── foo
└── bar
```

I'd like to be able to do this: 
```
>>> tree = Tree("Rich Tree", hide_root=True)
>>> tree.add("foo")
>>> tree.add("bar")
>>> print(tree)

foo
bar
```

Thanks for any guidance and also thanks for all the great work!!

**Comment by willmcgugan** [comment, marked as answer]
That would be a useful feature to add. Would you mind posting a feature request issue?

You can simulate it the moment by having a `Group` of trees for the top nodes.

**Reply by bgyarfas** [reply]
Will do. Thanks!
